armi/materials/material.py

```diff
@@ -45,26 +45,31 @@ class Material(composites.Leaf):
     A material is made up of elements or isotopes. It has bulk properties like mass density.
     """
 
+    # State parameter definitions
     pDefs = materialParameters.getMaterialParameterDefinitions()
-    """State parameter definitions"""
 
+    # Indication of where the material is loaded from (may be plugin name)
     DATA_SOURCE = "ARMI"
```

Comment:
this breaks sphinx rendering. Sphinx treats those docstrings as special and renders them nicely. see built docs. Undo or justify. 

see for example https://terrapower.github.io/armi/.apidocs/armi.materials.material.html#armi.materials.material.Material.DATA_SOURCE

Reply:
Eww. I don't like that at all.

But sure, I'll fix it back. Thanks!